src/Slide.vue

```diff
@@ -70,7 +70,7 @@ export default {
   },
   methods: {
     onTouchEnd(e) {
-      const eventPosX = this.carousel.isTouch
+      const eventPosX = this.carousel.isTouch && e.changedTouches && e.changedTouches.length > 0
```

Comment:
To be consistent with the rest of the component changing to *this.carousel.isTouch* is enough